### DISCUSSION TITLE
How to change main languge English to Japanese

### DISCUSSION THREAD

**Discussion by puraminy** [MrGreensWorkshop/MrGreen-JekyllTheme#88] [How to] [answered]
I cloned your project and changed main.yml to include only `ja` language as defalut with no langauge list. However the side nav shows two home and other items, one without ja and with it. 
I don't want any link to english pages and just ja folder, and it's the expected behaviour

I attached a screen shot of duplicate entries

![aaaa](https://user-images.githubusercontent.com/5293185/178716363-0067ae64-fd1e-4ccf-8ce6-18e73efb52fe.png)

**Comment by MrGreensWorkshop** [comment, marked as answer]
yes it is expected behavior. To change main language to japanese and remove english,
just move all of file and folders inside jp folder to outside of jp folder. overwrite all.

**Comment by puraminy** [comment]
yes, thanks, it resolved the problem
